style.css

```diff
@@ -0,0 +1,157 @@
+@media only screen and (max-width: 720px) {
+  @import url('https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Poppins:ital,wght@0,200;0,300;0,400;0,600;1,100&display=swap');
+
+  * {
+    background: #191765;
+    margin: 0;
+    padding: 0;
+  }
+
+  .navbar {
+    display: flex;
+    margin-top: 60px;
+    position: fixed;
+    top: 0;
+    gap: 196.33px;
+    width: 100%;
+    align-items: center;
+  }
+
+  .navname {
+    margin: 0 0 0 24px;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #fff;
+  }
```

Comment:
- [ ] [OPTIONAL] Please remove the margin or padding at the top of the Toolbar!

![image](https://user-images.githubusercontent.com/63915024/185445240-f16058d2-d251-4041-9a52-f728e94b4ea8.png)